actions/nuxeo-add-to-collection-button.html

```diff
@@ -157,13 +157,31 @@ <h2>[[i18n('addToCollectionButton.dialog.heading')]]</h2>
                 return this._newEntryFormatter.bind(this);
               },
             },
+
+            /**
+             * `true` if the action should display the label, `false` otherwise.
+             */
+            showLabel: {
+              type: Boolean,
+              reflectToAttribute: true,
+              value: false,
+            },
+
+            _label: {
+              type: String,
+              computed: '_computeLabel(i18n)',
+            },
           };
         }
 
         _isAvailable(doc) {
           return this.isCollectionMember(doc);
         }
 
+        _computeLabel() {
+          return this.i18n('addToCollectionButton.tooltip');
+        }
+
         _toggleDialog() {
           this.$['add-to-collection-dialog'].toggle();
```

Comment:
nit: guess we should've gone with JS friendly ids ;)

Reply:
I guess so :)